esphome/components/esp32_can/canbus.py

```diff
@@ -23,6 +25,8 @@
 CODEOWNERS = ["@Sympatron"]
 DEPENDENCIES = ["esp32"]
 
+CONF_TX_ENQUEUE_TIMEOUT = 'tx_enqueue_timeout'
```

Comment:
```suggestion
CONF_TX_ENQUEUE_TIMEOUT = "tx_enqueue_timeout"
```
Just keeping consistent style. Good choice for the name BTW.